### ISSUE TITLE
[Feature/Regression] Allow setting 'pdfjs.disabled: false' in Firefox

### ISSUE THREAD

**Issue by jhyot** [microsoft/playwright#23926]
We would like to enable the Firefox embedded PDF viewer. It worked before, but has been disabled by the developers. That's why I tagged with "Feature/Regression".

https://github.com/microsoft/playwright/issues/13157#issuecomment-1083481866 mentions that PDF testing is not in scope; but we don't want to test the PDF, but our site displays different content depending on whether the PDF can be viewed inline, and we would like to test the scenario of inline PDF.

https://github.com/microsoft/playwright/issues/13157#issuecomment-1082442487 mentions that the PDF viewer is explicitly disabled and can be enabled by using `firefoxUserPrefs`.

This actually does not work, as also mentioned on that thread. @rwoll [commented](https://github.com/microsoft/playwright/issues/13157#issuecomment-1083481866) that it should be used with `launchPersistentContext`, which cannot be true because that API does not even accept an `firefoxUserPrefs` config key.

If I followed various leads correctly, I think the real reason why setting that key on `firefoxUserPrefs` does not work, is because it is hard-codedly overwritten *after* the user prefs by https://github.com/microsoft/playwright/blob/2808f3700ac946c59d3fb954b3ceb5640c5a71ed/browser_patches/firefox/preferences/playwright.cfg#L12C30-L12C30

Can you make it possible to overwrite the hardcoded `pdfjs.disabled` key?
I don't think that that key qualifies for the comment in the source file stating `// THESE ARE THE PROPERTIES THAT MUST BE ENABLED FOR JUGGLER TO WORK`.

There might be a workaround by setting the prefs in the real config file in a persistent context (haven't tested it yet), but that greatly inconveniences the whole testing setup and I would like to avoid it.

**Comment by aslushnikov** [comment]
@jhyot This is a dupe of https://github.com/microsoft/playwright/issues/15405 that has been fixed in 1.34. I just checked and the following worked for me on Playwright 1.35:

```js
import { firefox } from '@playwright/test';

const browser = await firefox.launch({
  headless: false,
  firefoxUserPrefs: {
    'pdfjs.disabled': false,
  },
});
const page = await browser.newPage();
await page.goto('https://www.orimi.com/pdf-test.pdf');
```

**Comment by Ransely-Nigrel** [comment]
> @jhyot This is a dupe of #15405 that has been fixed in 1.34. I just checked and the following worked for me on Playwright 1.35:
> 
> ```js
> import { firefox } from '@playwright/test';
> 
> const browser = await firefox.launch({
>   headless: false,
>   firefoxUserPrefs: {
>     'pdfjs.disabled': false,
>   },
> });
> const page = await browser.newPage();
> await page.goto('https://www.orimi.com/pdf-test.pdf');
> ```

How can we achieve this in webkit ?
